src/main/java/net/openhft/chronicle/threads/DiskSpaceMonitor.java

```diff
@@ -145,14 +154,11 @@ void run() throws IOException {
             long unallocatedBytes = fileStore.getUnallocatedSpace();
```

Comment:
Maybe we should also move these checks to the strategy.

Reply:
Yes that sounds fair